src/RefractSourceMap.cc

```diff
@@ -0,0 +1,11 @@
+#include "RefractSourceMap.h"
+
+refract::IElement* drafter::BytesRangeToRefract(const mdp::BytesRange& bytesRange)
```

Comment:
What is the difference between this and `SourceMapRowToRefract` from `RefractParseResult.cc`? Infact, this is wrong since we need to use `CharactersRange`. Maybe we should move that function to here?